test/check-requirements.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+let reqs = require('../requirements.json');
+let request = require("request");
+
+let minSizes = new Object();
+let data = new Object();
+let fileNames = new Object();
+let count = 0;
+
+function checkRequirements() {
+  for (var attribute in reqs) {
+    data[attribute] = reqs[attribute].url;
+    count++;
+  }
+
+  //to check if the url looks like it points to what it is supposed to
+  fileNames['cdk.zip'] = 'cdk';
+  fileNames['rhel-vagrant-virtualbox.box'] = 'rhel-cdk';
+  fileNames['oc.zip'] = 'origin-client-tools';
+  fileNames['pscp.exe'] = 'pscp.exe';
+  fileNames['cygwin.exe'] = 'cygwin';
+  fileNames['jbds.jar'] = 'devstudio';
+  fileNames['jdk.zip'] = 'openjdk';
+  fileNames['vagrant.msi'] = 'vagrant';
+  fileNames['virtualbox.exe'] = 'virtualbox';
+
+  //to check if the files are rougly the size the should be
+  minSizes['cdk.zip'] = 50 * 1024;
+  minSizes['rhel-vagrant-virtualbox.box'] = 800 * 1024 * 1024;
+  minSizes['oc.zip'] = 10 * 1024 * 1024;
+  minSizes['pscp.exe'] = 300 * 1024;
+  minSizes['cygwin.exe'] = 500 * 1024;
+  minSizes['jbds.jar'] = 400 * 1024 * 1024;
+  minSizes['jdk.zip'] = 50 * 1024 *1024;
+  minSizes['vagrant.msi'] = 120 * 1024 * 1024;
+  minSizes['virtualbox.exe'] = 100 * 1024 * 1024;
+
+  console.log('-------------------------------');
+  console.log('Checking download URLs');
+  for (var key in data) {
+    checkFileName(key);
+    checkUrl(key);
+  }
+}
+
+function checkFileName(key) {
+  if (!data[key].includes(fileNames[key])) {
+    throw new Error(key + ' - the url does not contain ' + fileNames[key] + ': ' + data[key]);
+  }
+}
+
+function checkUrl(key) {
+  let req = request.get(data[key])
+  .on('response', function(response) {
+    let size = response.headers['content-length'];
+    if (response.statusCode !== 200) {
```

Comment:
Both virtualbox and vagrant use redirects, something like this happens:
virtualbox.exe - url: http://developers.redhat.com/redirect/to/virtualbox-5.0.8.download
virtualbox.exe - size: 117095112B